app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -182,11 +182,11 @@ You have the right to apply to train to teach in England if you are resident in
 * [the Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme)
 * [the Ukraine Extension Scheme](https://www.gov.uk/guidance/apply-to-stay-in-the-uk-under-the-ukraine-extension-scheme)
 
-### Deadlines for applications
+### Deadline for applications
 
-Applications to train to teach in 2022/23 have now closed. Applications for 2023/24 open on 11 October 2022.
+You can apply for teacher training courses starting in autumn 2023 until [September 2023](https://www.apply-for-teacher-training.service.gov.uk/provider/service-guidance/dates-and-deadlines).
 
-If you missed the deadline to apply this year, you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience for your teacher training course. 
+If you intend to [apply](https://www.gov.uk/apply-for-teacher-training), you could consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience until your teacher training course starts. 
```

Comment:
```suggestion
You could also consider finding work as a [teaching assistant](https://nationalcareers.service.gov.uk/job-profiles/teaching-assistant) to gain valuable experience until your teacher training course starts. 
```
We should avoid using one word links for accessibility reasons, so I've moved the apply link to the above sentence instead.